tools/release-image.sh

```diff
@@ -42,12 +42,4 @@ if [ -z "${t}" ]; then
 fi
 
 cd images/$1/
-docker build -t prairielearn/$1 .
-
-if [ $? -ne 0 ]; then
-  echo "ERR: building image failed. skipping upload." >& 2
-  exit 4
-fi
-
-docker tag prairielearn/$1 prairielearn/$1:${t}
-docker push prairielearn/$1:${t}
+docker buildx build --push --platform linux/arm64/v8,linux/amd64 -t prairielearn/$1:${t} .
```

Comment:
`buildx` is not able to load multi-arch images into the local docker instance (see [here](https://docs.docker.com/engine/reference/commandline/buildx_build/#output)), so this builds and pushes in one step. I don't know if that will break anything.

Reply:
This should be fine.